geowebcache/s3storage/src/main/java/org/geowebcache/s3/S3Ops.java

```diff
@@ -414,7 +414,7 @@ public Long call() throws Exception {
                         String.format(
                                 "S3 bulk delete aborted for '%s/%s'. Will resume on next startup.",
                                 bucketName, prefix));
-                return null;
+                throw e;
```

Comment:
Perhaps this is whats causing the AppVeyor test failure? Did you check how null results were handled from where this gets called?

Reply:
It seems to be one of the same errors occurring on other PRs